coursera-dl.bat

```diff
@@ -6,4 +6,9 @@ rem Usage: see "Running the script" section of the README for sample commands.
 
 rem Run "coursera-dl" in the current folder as working directory (where "%~dp0" is the
 rem location of this .bat file) via Python, passing all user-specified arguments ("%*")
-python %~dp0\coursera-dl %*
+
+rem This command does not work on Windows paths with spaces
+rem python %~dp0\coursera-dl %*
+
+rem Changed to this command. That does work with spaces in path. YAY! (Victor Westmann 2015-11-09 1:01 AM)
```

Comment:
I know about your excitement, but git is excellent at tracking changes, authorship and dates and you can cut off that part from the changes (there's no need to cancel this pull request---just change the file, add another commit and push it; it will "magically" appear here to me).